python/paddle/nn/functional/activation.py

```diff
@@ -49,12 +49,13 @@ def celu(x, alpha=1.0, name=None):
     Examples:
         .. code-block:: python
 
-            import paddle
-            import paddle.nn.functional as F
-            x = paddle.to_tensor([[-1., 6.], [1., 15.6]])
-            out = F.celu(x, alpha=0.2)
-            # [[-0.19865242,  6.        ],
-            #  [ 1.        , 15.60000038]]
+            >>> import paddle
+            >>> import paddle.nn.functional as F
+
+            >>> x = paddle.to_tensor([[-1., 6.], [1., 15.6]])
+            >>> out = F.celu(x, alpha=0.2)
+                [[-0.19865242,  6.        ],
+                [ 1.        , 15.60000038]]
```

Comment:
```suggestion
            >>> x = paddle.to_tensor([[-1., 6.], [1., 15.6]])
            >>> out = F.celu(x, alpha=0.2)
            >>> print(out)
            [[-0.19865242,  6.        ],
             [ 1.        , 15.60000038]]
```

- 代码示例如果有输出，应该加 `print()`  或者直接写变量名 `out`

- 输出的内容的缩进应该和 `>>>` 对齐